aiida_vasp/utils/fixtures/calcs.py

```diff
@@ -9,11 +9,13 @@
 def vasp_calc_and_ref(vasp_code, vasp_params, paws, vasp_kpoints, vasp_structure, ref_incar):
     """Fixture for non varying setup of a vasp calculation"""
     from aiida_vasp.calcs.vasp import VaspCalculation
+    from aiida.orm.data.parameter import ParameterData
     calc = VaspCalculation()
     calc.use_code(vasp_code)
     calc.set_computer(vasp_code.get_computer())
     calc.set_resources({'num_machines': 1, 'num_mpiprocs_per_machine': 1})
     calc.use_parameters(vasp_params)
+    calc.use_settings(ParameterData(dict={'parser_settings': {'add_bands': True, 'add_dos': True}}))
```

Comment:
It doesn't have to be a fixture if it's not used in any other place.

Reply:
Noted. Since this is currently the only test using 'settings'. This might change later in case that the VaspParser would end up to be generic enough that it can also manage other parsers. But that is not the case yet.